docs/frameworks/symfony.md

```diff
@@ -56,7 +58,11 @@ functions:
 
 We still have a few modifications to do on the application to make it compatible with AWS Lambda.
 
-Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache and the logs are stored in the `src/Kernel.php` file. This is done by adding 2 new methods to the `Kernel` class:
+Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache and the logs are stored in the `src/Kernel.php` file. This is done by adding 3 new methods to the `Kernel` class. 
+We still want our [build directory](https://symfony.com/doc/current/reference/configuration/kernel.html#build-directory) 
```

Comment:
Maybe something like: 
```suggestion
To be able to deploy our application with a warm cache, we need to specify the [build directory](https://symfony.com/doc/current/reference/configuration/kernel.html#build-directory) to be separate from the cache directory.
```

Then explain that you need to warm the cache in the deploy process.